CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
-## Release 0.1.0 (2021-11-29)
```

Comment:
This release date is in the past?

And you add a change in this PR for something that would be in the release.

I don't understand how this is possible?

Reply:
The release never went trough since the Integration test is blocking it

Reply:
Oh well, turns out I read this diff in the opposite way.
I thought you added a release date, but in fact you removed it.

My bad.